pkg/kubectl/cmd/set/set_subject.go

```diff
@@ -262,7 +262,9 @@ func (o *SubjectOptions) Run(fn updateSubjects) error {
 			continue
 		}
 
-		return o.PrintObj(actual, o.Out)
+		if err := o.PrintObj(actual, o.Out); err != nil {
+			return err
```

Comment:
Why not:
```
allErrs = append(allErrs, err)
```

After this change `kubectl set` will have the same behavior with kubectl 1.10, right?

Reply:
@CaoShuFeng We got couple of similar process in other files. I think we'd better keep same pace  with them.For example:
https://github.com/kubernetes/kubernetes/blob/23d9a48e6a59bc3e4faa84a7cf1416853521e60c/pkg/kubectl/cmd/set/set_image.go#L291

Reply:
agree on aggregating errors rather than short-circuiting